modules/all-modules/pom.xml

```diff
@@ -118,11 +118,6 @@
         </dependency>
         <dependency>
             <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-file-extension-common</artifactId>
```

Comment:
I agree, I already created an issue for having a parent pom for this kind of artifacts, perhaps could be a new group of artifacts at mule/plugins/file-extension-common.